docs/9-faq.md

```diff
@@ -1,13 +1,18 @@
 # FAQ
 
-## Do I have to use [Homebrew](https://brew.sh/) to install Zarf?
+## Do I have to use Homebrew to install Zarf?
```

Comment:
Any particular reason (besides style preference) to remove the link to homebrew?

Reply:
Having a link in a header is usually frowned upon. If we want to link to homebrew i think we should include it in the description below. 

However, if a user is asking if they can use something other than homebrew, I think we can assume that the user is not interested int he homebrew link at that time.